tests/utils/streams/test_kafka.py

```diff
@@ -67,11 +68,16 @@ def test_consumer(topic: str) -> None:
     message = consumer.poll(1.0)
     assert isinstance(message, Message)
     assert message.stream == TopicPartition(topic, 0)
+    assert message.offset == 0
     assert message.value == value
 
-    start = time.time()
-    assert consumer.poll(0.0) is None
-    assert time.time() - start < 0.001  # consumer should not block
```

Comment:
I realized this wasn't actually deterministic before: it was possible that it would raise a `EndOfStream` error. (I guess the current one isn't technically deterministic either, hopefully but you get the idea.)